src/core/stats/remote-metrics.ts

```diff
@@ -72,8 +73,14 @@ export class RemoteMetrics {
     }, {} as Record<string, string>)
 
     formatted['library'] = 'analytics.js'
-    formatted['library_version'] =
-      `next-${process.env.VERSION}` ?? 'next-undefined'
+
+    if (typeof process.env.VERSION !== 'undefined') {
```

Comment:
Would reading `process.env.VERSION` create a similar issue to https://github.com/segmentio/analytics-next/issues/303 for non-webpack users? I think if `process.env` is undefined this may cause exception 🤔

Reply:
Also in order to follow DRY ( we have this check in two places), might want to consolidate this logic somewhere, like a `getVersion` method that does this check and returns `npm` vs. `web` versions.

Reply:
typeof will return 'undefined' if the variable doesn't exist, not an exception

Reply:
I think for those users, the `process` is undefined, so `typeof process.env.VERSION` will throw a 
`Uncaught ReferenceError: process is not defined`